packages/fast-components-react-base/app/examples.ts

```diff
@@ -1,2 +1,5 @@
 import ButtonExamples from "../src/button/examples.data";
 export {ButtonExamples};
+
+import ToggleExamples from "../src/toggle/examples.data";
+export {ToggleExamples};
```

Comment:
Is this not yelling at you for a newline?

Reply:
Nah, I can add it back in though. I received maybe one or two warnings about this at different times, but it seems all over the place when it "cares"

Reply:
This would get caught on CI as well.